features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -542,6 +543,24 @@ class InfoBar extends React.Component {
         }
     };
 
+    getSchema() {
+        const newAPI = new API();
+        const { apiId } = this.props;
+        const { api } = this.context;
+        const promisedGraphQL = newAPI.getGraphQLSchemaByAPIId(apiId);
+        promisedGraphQL.then((response) => {
+            const windowUrl = window.URL || window.webkitURL;
+            const binary = new Blob([response.data]);
+            const url = windowUrl.createObjectURL(binary);
+            const anchor = document.createElement('a');
+            anchor.href = url;
+            anchor.download = api.provider + '-' + api.name + '-' + api.version
+            + '.graphql';
```

Comment:
can go in one line

Reply:
Fixed with ead77f9051a2e0c270afb84bd6c21f3263995bbe